glusterd2/snapshot/store-utils.go

```diff
@@ -112,7 +113,27 @@ func GetSnapshot(name string) (*Snapinfo, error) {
 
 //DeleteSnapshot passes the snap path to store to delete the snap object
 func DeleteSnapshot(snapInfo *Snapinfo) error {
+	var vol *volume.Volinfo
 	_, e := gdstore.Store.Delete(context.TODO(), GetStorePath(snapInfo))
```

Comment:
store wrapper has been introduced in PR #857 
the code should be changed as 
```
_, e := gdstore.Delete(context.TODO(), GetStorePath(snapInfo))
```

Reply:
done